### ISSUE TITLE
Better sorting of entries

### ISSUE THREAD

**Issue by SkyyySi** [ibraheemdev/modern-unix#27]
Instead of just dropping in all tools in a (seemingly) random order, the entries should be sorted by A) a category and maybe also B) alphabetically (per-category). This could be further improved by adding a table of contents at the top.

**Comment by ibraheemdev** [comment]
Yeah, I've been wanting to do this, but I'm not sure of the best way. I want to highlight the top tools, but not exclude other alternatives that may be slightly different. I'm not even sure if I want to include TermGUIs or system monitors, because they aren't really "unix command replacements".

**Comment by fcolecumberri** [comment]
you mean like in https://github.com/johnalanwoods/maintained-modern-unix ?
